navi/src/main/java/com/trello/navi/model/ViewCreated.java

```diff
@@ -0,0 +1,39 @@
+package com.trello.navi.model;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.View;
+
+public class ViewCreated {
+    private final Bundle bundle;
+    private final View view;
+    public ViewCreated(View view, @Nullable Bundle bundle) {
+        this.view = view;
+        this.bundle = bundle;
+    }
+    @Nullable
+    public Bundle getBundle() {
+        return bundle;
+    }
+    public View getView() {
```

Comment:
Should just be `bundle()` and `view()`, to match the other models.

Also should be ordered consistently throughout - always view then bundle.